src/helpers/ConnectorConfigInput.tsx

```
@@ -137,7 +137,7 @@ class ConnectorConfigInput {
     if (exclude.length > 0) {
       if (exclude[0] in data?.properties) {
```

Biome (2.4.4)

[error] Unsafe usage of optional chaining.

(lint/correctness/noUnsafeOptionalChaining)